src/interfaces/vault.ts

```diff
@@ -177,8 +177,25 @@ export class VaultInterface<T extends ChainId> extends ServiceInterface<T> {
         return vaultContract.deposit(amount, overrides);
       }
     } else {
-      // ZAPPER!
-      throw new SdkError("deposit:zapper not implemented.");
+      const zapInParams = await this.yearn.services.zapper.zapIn(
+        account,
+        token,
+        amount,
+        vault,
+        overrides.gasPrice?.toString() || "0",
+        0.01
+      );
+
+      const transaction: TransactionRequest = {
+        to: zapInParams.to,
+        from: zapInParams.from,
+        gasLimit: overrides.gasLimit?.toString() || "0",
```

Comment:
should I remove the ` || "0"` not sure where we're gonna get the `gasLimit` from